### PR TITLE
Remove tons of dead or game code.

### PR DESCRIPTION
**Pull request by PJB3005**

### CODE REVIEW

File: SS14.Shared/Class1.cs [hunk 1]
```diff
@@ -1,11 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-
-namespace SS3D_shared
-{
-    public class Class1
```
Comment (PJB3005) [inline SS14.Shared/Class1.cs:8]: This file is particularly cute.

**Comment by BobdaBiscuit** [comment]
WE'RE NOT GONNA HAVE ANYTHING LEFT

**Comment by ComicIronic** [comment]
Maybe we could spin this out into a game content repo? There's some decent emulation of current mechanics that I don't feel like doing twice for no reason. 6000 lines of code is >1/10th of all the stuff we have.

**Comment by PJB3005** [comment]
@ComicIronic we can re-add that later. A lot of this code is pretty awful either way though and probably is better off rewriting from scratch (job and ban code didn't even use IoC, etc...)

There's always the git history.

**Comment by psykzz** [comment]
We could always git tag this part, or keep a branch of pre-content-removal

**Comment by psykzz** [comment]
@PJB3005 
Conflicting files
SS14.Server.Interfaces/Crafting/ICraftingManager.cs
SS14.Server.Services/Configuration/ConfigurationManager.cs
SS14.Server.Services/Crafting/CraftingManager.cs
SS14.Server.Services/MessageLogging/MessageLogger.cs

**Comment by ZoldorfTheWizard** [comment]
@PJB3005 fix pls

**Pull request merged**